### DISCUSSION TITLE
My project keeps crashing and making pink squares

### DISCUSSION THREAD

**Discussion by LoneStarz** [keiwando/toonsquid#21] [Q&A] [answered]
Hello so I’ve been having this problem with this project that makes the app crash and turns the Frame I was working on a pink square.

![IMG_3852](https://user-images.githubusercontent.com/115443798/196004746-85cb5fc6-22fc-4031-b4b3-b0b473b47dea.png) 

I’m not sure if it’s because it’s too intense for the app to handle? I’m running on iPad Pro 2018 11”  (iOS 16 beta)

**Comment by keiwando** [comment, marked as answer]
Hi! I'm sorry about that! Would it be possible for you to export and send me the project via contact@toonsquid.com so that I can take a look at what exactly might be going on?

**Reply by LoneStarz** [reply]
Thank you so much for your answer! apparently the problem was with memory leak when undoing and he’s releasing an update to fix the problem soon!. Thank you for being so quick with the solution!

**Reply by keiwando** [reply]
Just for future reference: The memory leak that ultimately led to a crash after many times of undoing a fill layer edit was fixed with 1.5.3 and the issue of potentially losing previous pixel layer contents after edits if the editor crashes before the next auto save (the issue with the pink rectangles) should be fixed with the 1.5.4 update which is already submitted and will hopefully be available on the App Store in the coming days.

**Comment by trinhpxl** [comment]
your animation looks amazimg so far ! do you plan to share it around here when done ? :D

**Reply by LoneStarz** [reply]
Thank you!. Yes! I Plan of sharing it soon on my YouTube channel!!
